### PR TITLE
Add script to simplify running longhorn command

### PR DESCRIPTION
**Pull request by shuo-wu**
https://github.com/longhorn/longhorn/issues/655

### CODE REVIEW

**Comment by shuo-wu** [comment]
Not sure which directory the script should be put in. Currently, it's in the root directory of the project.

**Comment by yasker** [comment]
> Not sure which directory the script should be put in. Currently, it's in the root directory of the project.

Put it in `deploy/scripts`.

File: longhorn-cmd [hunk 1]
```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env bash
+
+cat << EOF
+USAGE:
+  # Need to use quotation marks on [longhorn-command-args] as the 2nd arg of this script
```
Comment (yasker) [inline longhorn-cmd]: `-` should work 

Also, must have longhorn installed in `longhorn-system` namespace and have access to `kubectl` and the namespace

Comment (yasker) [reply longhorn-cmd]: User shouldn't need to add any extra characters for this to work.

**Comment by shuo-wu** [comment]
I have added executable permission for this script before pushing it into github.

**Comment by yasker** [comment]
@shuo-wu Wrong directory for the script.

**Pull request merged**